selftests/check.py

```diff
@@ -604,4 +635,5 @@ def main():
 
 
 if __name__ == '__main__':
-    sys.exit(main())
+    args = parse_args()
```

Comment:
I'm curious: any reason for this change?

Reply:
Yes, if you look at the remaining changes, there is a call to `selftests.check.main(args)` from `setup.py`
Please, tell me if you know of a better way to do this.

Reply:
> Yes, if you look at the remaining changes, there is a call to `selftests.check.main(args)` from `setup.py`

Oh, I missed this.  Thanks!

> Please, tell me if you know of a better way to do this.

Nope, that's fine!